src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php

```diff
@@ -62,11 +68,7 @@ public function getContainer($name, array $parameters = [], $createIfNotExists =
             $widgetModel = $containerWidget->getWidget();
             $widgetClass = $widgetModel->getType();
             $widgetHandler = new $widgetClass($widgetModel);
-            if ($widgetHandler instanceof ContainerAwareInterface) {
```

Comment:
Rafal asked me to take it out

On Thu, Jun 30, 2016 at 12:50 PM, Paweł Mikołajczuk <
notifications@github.com> wrote:

> In src/SWP/Bundle/TemplateEngineBundle/Service/ContainerService.php
> <https://github.com/superdesk/web-publisher/pull/79#discussion_r69112105>:
>
> > @@ -62,11 +68,7 @@ public function getContainer($name, array $parameters = [], $createIfNotExists =
> >              $widgetModel = $containerWidget->getWidget();
> >              $widgetClass = $widgetModel->getType();
> >              $widgetHandler = new $widgetClass($widgetModel);
> > -            if ($widgetHandler instanceof ContainerAwareInterface) {
>
> Why you don't check now if interface is used?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/superdesk/web-publisher/pull/79/files/c37ce8f10848942de9b79e03a6cc3a03c30ad777..b3d47863897fb0618676e5da8f913872d908cabb#r69112105>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARBtFQzl1tLnZfATeb5eue1sydUNHainks5qQ59pgaJpZM4I2oYm>
> .
>